src/SignalR.Orleans/Core/ConnectionGroupGrain.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Orleans;
+using Orleans.Streams;
+using SignalR.Orleans.Clients;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SignalR.Orleans.Core
+{
+    internal abstract class ConnectionGroupGrain<TGrainState> : Grain<TGrainState>, IConnectionGroupGrain where TGrainState : ConnectionGroupState, new()
```

Comment:
I'm not quite convinced of the _Group_ in the name... It can confuses with Group grain. Will think about it...

Reply:
I will rename it to `ConnectionGrain` what do you think? I will also rename the methods to: `AddConnection` and `RemoveConnection`.

Reply:
done